### PR TITLE
12.0 mig report qweb parameter

### PR DESCRIPTION
**Pull request by ValentinVinagre**
Standard migration to 12.0

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Can you check Travis status?

File: report_qweb_parameter/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```
Comment (etobella) [inline report_qweb_parameter/tests/__init__.py:1]: No longer required (it happens on all files)

**Comment by etobella** [comment]
Travis is red. @ValentinVinagre Can you review?

**Review by etobella — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
@ValentinVinagre are you going to fix Travis? There's other PR in #301

**Comment by pedrobaeza** [comment]
Superseeded by #301

**Pull request closed without merging**